docs/Documentation.md

```diff
@@ -1,29 +1,39 @@
----
-layout: page
-title: Documentation guide
----
+<frontmatter>
+  layout: default.md
+  title: "Documentation guide"
+  pageNav: 3
+  pageNavTitle: "Table Of Content"
+</frontmatter>
+
+# Documentation guide
 
 **Setting up and maintaining the project website:**
 
-* We use [**Jekyll**](https://jekyllrb.com/) to manage documentation.
+* We use [**MarkBind**](https://markbind.org/) to manage documentation.
 * The `docs/` folder is used for documentation.
-* To learn how set it up and maintain the project website, follow the guide [_[se-edu/guides] **Using Jekyll for project documentation**_](https://se-education.org/guides/tutorials/jekyll.html).
+* To learn how set it up and maintain the project website, follow the [_**MarkBind User Guide**_](https://markbind.org/userGuide/gettingStarted.html).
+  * Quick start:
+    * `cd docs # go to the docs folder`
+    * `npm i # install dependencies, only needed once when setting up the project`
+    * `npm run serve # preview the website locally and make edits`
```

Comment:
How about `npm run serve -d` for hot-reloading?

Reply:
as communicated.